drift/drift-idl-generator/src/main/java/com/facebook/drift/idl/generator/ThriftIdlRenderer.java

```diff
@@ -162,16 +162,39 @@ private String renderStruct(ThriftStructMetadata struct)
             }
             builder.append(separator.getAndUpdate(!field.getDocumentation().isEmpty()))
                     .append(documentation(field.getDocumentation(), "  "))
-                    .append(format("  %s: %s%s %s;\n",
+                    .append(format("  %s: %s%s %s%s;\n",
                             field.getId(),
                             requiredness(field.getRequiredness()),
                             typeName(field.getThriftType()),
-                            field.getName()));
+                            field.getName(),
+                            formatMap(field.getIdlAnnotations())));
         }
 
         return builder.append("}\n").toString();
     }
 
+    private static String formatMap(Map<String, String> map)
```

Comment:
nit: Is Guava available in this module? If it is consider using Map joiner: https://gist.github.com/K0NRAD/8898732#file-mapjoinertest-java